plugins/catalog/src/plugin.ts

```diff
@@ -82,7 +82,7 @@ export const catalogPlugin = createPlugin({
     options?: CatalogInputPluginOptions,
   ): CatalogPluginOptions {
     const defaultOptions = {
-      createButtonTitle: 'Create',
+      createButtonTitle: 'Create component',
```

Comment:
hmm technically you _can_ create users/groups if you source them from catalog spec files.

The create button title is also configurable so maybe it makes more sense to customize this for your instance instead of changing the default?

Reply:
Yeah I also though that too but I don't think you can by default, you have to change the rules for that. But to be honest I agree that you can override this how you see fit and maybe it's not worth the updating of the default one when you can configure it anyways.